.gitattributes

```diff
@@ -0,0 +1 @@
+* text eol=lf
```

Comment:
You probably want to add some directives to avoid modifying binary files, like

```
# Denote all files that are truly binary and should not be modified.
*.png binary
*.jpg binary
```